app/views/linkers/index.html.erb

```diff
@@ -48,25 +48,26 @@ function readCSVFile() {
 
     var lines = reader.result.split('\n');
 
-    for(var i = 0; i < lines.length; i+=1) {
+    lines.forEach(function(line) {
```

Comment:
You need to do use a forEach to encapsulate the scope, or else your request will get overridden on each iteration through the for loop.